terraform/fargate-task-definition/_variables.tf

```diff
@@ -0,0 +1,88 @@
+variable "name" {
+  description = "The service name"
+  type        = string
+}
+
+variable "cpu" {
+  description = "Number of CPU units"
+  type        = number
+}
+
+variable "memory" {
+  description = "Memory MB"
+  type        = number
+}
+
+variable "env_vars" {
+  description = "Key/value map of environment variables for the task"
+  type        = map(string)
+  default     = {}
+}
+
+variable "secrets" {
+  description = "Key/value map of the secrets"
+  sensitive   = true
+  default     = {}
+  validation {
+    condition     = length([for k, v in var.secrets : k if v == ""]) == 0
+    error_message = "Values for secrets cannot be empty. To pass an empty string, set it as an env var."
+  }
+}
+
+variable "ports" {
+  description = "The ports to open on the container"
+  type        = list(number)
+  default     = []
+}
+
+variable "image" {
+  description = "The Docker image to pull"
+}
+
+variable "docker_labels" {
+  description = "Key value map of docker labels"
+  type        = map(string)
+  default     = {}
+}
+
+variable "command" {
+  description = "The command to run"
+  default     = null
+  type        = list(string)
+}
+
+variable "env" {
+  description = "Environment name"
+}
+
+variable "additional_task_role_statements" {
+  type = list(object({
+    Effect   = string
+    Action   = list(string)
+    Resource = list(string)
+  }))
+  default = []
+}
+
+variable "efs_mounts" {
```

Comment:
Truthfully I'm unsure, and it may be time consuming to verify. I'd rather leave this in for now for consistency with the existing Ephemera configuration, at least until the whole system is running

Reply:
Think we can ditch this. Was specifically added for the message history server (which won't get deployed here), and even that is going to stop using the mounts.